File: test_read_data.r
library(tictoc)
library(arrow)
library(tidyverse)

# Read csv file using pandas and get time of read file in [s]
start_time = tic()
df_1 <- read_csv("sample.csv")
stop_time = toc()
print(paste('The csv read took ',gsub(" sec elapsed","",stop_time$callback_msg),'seconds'))

# Read parquet file using pandas and get time of read file in [s]
start_time = tic()
df_2 <- read_parquet("sample.parquet")
stop_time = toc()
print(paste('The parquet read took ',gsub(" sec elapsed","",stop_time$callback_msg),'seconds'))

File: test_read_data.py
import pandas as pd
import time

# Read csv file using pandas and get time of read file in [s]
start_time = time.time()
df_1 = pd.read_csv('sample.csv')
stop_time = time.time()
print('The csv read took %s seconds' %(stop_time - start_time))


# Read parquet file using pandas and get time of read file in [s]
start_time = time.time()
df_2 = pd.read_parquet('sample.parquet', engine='pyarrow')
stop_time = time.time()
print('The parquet read took %s seconds' %(stop_time - start_time))